src/memory.c

```diff
@@ -30,7 +34,20 @@ void cpuio_write(uint8_t reg, uint8_t value);
 void
 memory_init()
 {
+	//Initialize RAM array
 	RAM = calloc(RAM_SIZE, sizeof(uint8_t));
+	
+	//Initialize RAM access flag array
+	RAM_access_flags = (bool*) malloc(RAM_SIZE * sizeof(bool));
+	
+	//Randomize all RAM and set all access flags to false
+	time_t t;
+	srand((unsigned)time(&t));
+	for (int i = 0; i < RAM_SIZE; i++){
+		RAM[i] = rand();
```

Comment:
I'm not a fan of randomizing RAM. It makes RAM dumps much less readable. Is the uninitialized warning not enough?

If you want to keep it, I'd put it behind a second command line option.